shuup/admin/modules/orders/toolbar.py

```diff
@@ -72,50 +76,59 @@ def _build_order_set_state_button(self):
             )
 
     def _build_set_complete_button(self):
-        self.append(PostActionButton(
```

Comment:
you can remove this `_build_set_complete_button` completely. This button will be visible in the menu built in `_build_order_set_state_button` method, right?